internal/config/default_config.go

```diff
@@ -44,7 +44,7 @@ var (
 			// Core Resources
 			{
 				Name:      "replicasets.v1.apps",
-				PublishTo: "meshery.meshsyc.core",
+				PublishTo: "meshery.meshsync.core",
```

Comment:
Whoa!

How long has this been here, @Aritra8438? This would be causing MeshSync to completely fail to perform its job, I believe.